app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt

```diff
@@ -124,77 +120,98 @@ class OtherUserProfileScreenViewModel @Inject constructor(
 
     init {
         state = state.copy(isDataLoading = true)
+
+        observeUserInfoAndUpdateViewState()
+
+        viewModelScope.launch {
+            persistOtherUserClients(userId)
+        }
+    }
+
+    private fun observeUserInfoAndUpdateViewState() {
+        viewModelScope.launch {
+            observeUserInfo(userId)
+                .onEach {
```

Comment:
Maybe instead of filtering results make a fold in collect and update state or show snackbar?

Reply:
okay